api/redirects.go

```diff
@@ -48,12 +56,14 @@ func (api *RedirectAPI) getRedirect(w http.ResponseWriter, r *http.Request) {
 
 	redirectResponse, err := json.Marshal(responseBody)
 	if err != nil {
+		log.Error(ctx, "request failed", err, logData)
```

Comment:
Be more specific in log. Also don't return internal error details to client.
```suggestion
		log.Error(ctx, "request failed", err, logData)
		api.handleError(ctx, w, apierrors.ErrInternal, http.StatusInternalServerError)		
```

Reply:
👍